### PR TITLE
Brynjar H's card

### PR DESCRIPTION
**Pull request by binnih**
Added my card to the index.html file.

### CODE REVIEW

**Comment by ascodeasice** [comment]
Hi, you should replace "Your handle" in line 129 and 131 with your username.
And also update the href field in line 138 and in other anchor tags.

I'll review this PR again after you've made changes, and help you merge it.

File: index.html [hunk 1]
```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
...
+            <a href="https://www.twitter.com/BinnihH" target="_blank">BinnihH</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/binnih" target="_blank">binnih</a>
+          </p>
+          <p class="about">Learning to contribute to open source.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">Resource 1</a>
```
Comment (ascodeasice) [inline index.html:138]: You also need to change
1.  the href tag in every <a>
2. "Resource X" text
3. title in every <a>

such as
```suggestion
                <a href="https://www.theodinproject.com/" target="_blank" title="The Odin Project">The Odin Project</a>
```

**Comment by CandidDeer** [comment]
@binnih Please note that we will close any PRs with unresolved change requests 7 days after initial opening. Thx.

**Comment by ascodeasice** [comment]
Good job, I've merged your PR. You will be able to see your card on our website in a moment

**Pull request merged**